### PR TITLE
LL-8791 LL-9146 (Market): release

### PR DESCRIPTION
**Pull request by LFBarreto**
## 🦒 Context (issues, jira)

[LL-8791]
[LL-9146]

- activate Market tab
- add analytics to market tab
- removes shared code in favor of LLC common logic for market provider
- escapes potential issues on ipcRenderer malformed logging

- removes dusk mode and updates theme color (to confirm with design if ok)

## 💻  Description / Demo (image or video)

<details>
    <summary> :camera_flash:  </summary>
    
localhost_8080_webpack_index html_theme=dark
localhost_8080_webpack_index html_theme=light

</details>

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [ ] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

[LL-8791]: https://ledgerhq.atlassian.net/browse/LL-8791?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LL-9146]: https://ledgerhq.atlassian.net/browse/LL-9146?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/preloader/logo.svg [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+<svg width="80" height="80" xmlns="http://www.w3.org/2000/svg">
```
Comment (juan-cortes) [inline src/preloader/logo.svg:1]: My svg eyes tell me this is the old icon, is that relevant, or should it be the new thing?
image

File: src/renderer/components/Onboarding/Screens/Tutorial/screens/PinCodeHowTo.js [hunk 2]
```diff
@@ -49,8 +49,8 @@ const StepIndexContainer = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
-  background: rgba(100, 144, 241, 0.1);
-  color: #6490f1;
+  background: rgba(138, 128, 219, 0.1);
+  color: #8a80db;
```
Comment (juan-cortes) [inline src/renderer/components/Onboarding/Screens/Tutorial/screens/PinCodeHowTo.js:53]: What's up with all the color changes? 😕

Comment (juan-cortes) [reply src/renderer/components/Onboarding/Screens/Tutorial/screens/PinCodeHowTo.js:53]: Not that I care but the scope of the task is supposed to be activating the page + porting some logic

**Review by juan-cortes — COMMENTED** [review]
I don't feel comfortable approving without understanding why there are so many UI changes 😓

File: src/renderer/components/Onboarding/Screens/Tutorial/screens/PinCodeHowTo.js (hunk 2 above)
Comment (LFBarreto) [reply src/renderer/components/Onboarding/Screens/Tutorial/screens/PinCodeHowTo.js:53]: removes dusk mode and updates theme color `(to confirm with design if ok)`
we want to change the main color to match the new themes all other the app and not have discrepancy between market vs the rest.
the idea is ok with design expect the purple i put seems to be the wrong one :sweat_smile:

Comment (juan-cortes) [reply src/renderer/components/Onboarding/Screens/Tutorial/screens/PinCodeHowTo.js:53]: So then it's wrong Luiz, if the color is wrong, it's all wrong  (╯°□°)╯︵ ┻━┻

**Review by nabil-brn — APPROVED** [review]
Tested ✔️ 

---
cd43dc0b6 (HEAD -> LL-8791-market-analytics, ledger/LL-8791-market-analytics) (Market): fix swap link in coin details page + breadcrumb

**Pull request merged**